core/cas-server-core-web/src/main/java/org/apereo/cas/config/CasCoreWebConfiguration.java

```diff
@@ -30,6 +33,24 @@
     @Autowired
     private CasConfigurationProperties casProperties;
 
+    /**
+     * Get messages from one or more property files containing values
+     * that should be exposed to Thyme templates but don't need i18n.
+     * This bean is Autowired into {@link messageSource()}.
+     * @return PropertiesFactoryBean containing all common (non-i18n) messages
+     */
+    @Bean
+    public PropertiesFactoryBean casCommonMessages() {
```

Comment:
It's possible, although I am not sure what you mean by creating it internally. You can only set one properties object as the common messages so I was trying to use a PropertiesFactoryBean to set it up so that it would be easier for other properties files to be included later (I know that violates a principal of writing code for requirements that don't currently exist). If I manually make a PropertiesFactoryBean rather than letting Spring make it as a bean, then I have to invoke methods from the InitializingBean interface and the FactoryBean interface that spring does automatically. I thought making it a bean was a little cleaner. If I am only trying to support a single property file then it probably makes it cleaner to not create a bean and just set it directly on the message resources.